src/api/resize.js

```diff
@@ -50,27 +20,43 @@ async function parseArgs (args) {
     return args[1] === 'portrait' ?
            { width: size.portraitWidth, height: size.landscapeWidth } :
            { width: size.landscapeWidth, height: size.portraitWidth };
+
+}
+
+function getCorrectedSize (currentClientAreaSize, currentWindowSize, requestedSize) {
+    var horizontalChrome = currentWindowSize.width - currentClientAreaSize.width;
+    var verticalChrome   = currentWindowSize.height - currentClientAreaSize.height;
+
+    return {
+        width:  requestedSize.width + horizontalChrome,
+        height: requestedSize.height + verticalChrome
+    };
 }
 
 /**
- * Changes the browser window size to the new width and height.
+ * Changes the browser client area size to the new width and height.
  * @function
  * @async
  * @name resize
  * @param {string} pageUrl - Specifies the URL of the web page opened in the browser.
+ * @param {number} currentWidth - Specifies current width of browser client area. Actually it is the value of the `window.innerWidth` property.
+ * @param {number} currentHeight - Specifies current height of browser client area. Actually it is the value of the `window.innerHeight` property.
  * @param {number} width - Specifies the new window width in pixels.
  * @param {number} height - Specifies the new height in pixels.
  **/
 /**
- * Changes the browser window size according to the screen size of the target device.
+ * Changes the browser client area size according to the screen size of the target device.
```

Comment:
Changes the browser's client area size according to the screen size of the target device.